backend/models/error.py

```diff
@@ -0,0 +1,4 @@
+from pydantic import BaseModel
+
+class APIError(BaseModel):
+    detail: str
```

Comment:
Should define an enum as error code to know which type of errors for controller to handle

Reply:
> Should define an enum as error code to know which type of errors for controller to handle

I will let it as a future feature as I don't know how to implement error handler in a correct way.

Reply:
You can create an issue and we will work on this